saleor_gql_loader/data_loader.py

```diff
@@ -65,6 +65,117 @@ def __init__(self, auth_token, endpoint_url="http://localhost:8000/graphql/"):
         self.headers = {"Authorization": "Bearer {}".format(auth_token)}
         self.endpoint_url = endpoint_url
 
+    def update_shop_settings(self, **kwargs):
+        """update shop settings.
+
+        Parameters
+        ----------
+        **kwargs : dict, optional
+            overrides the default value set to update the shop settings refer to the
+            ShopSettingsInput graphQL type to know what can be overriden.
+
+        Raises
+        ------
+        Exception
+            when shopErrors is not an empty list
+        """
+
+        variables = {
+            "shopSettingsInput": kwargs
+        }
+
+        query = """
+            mutation ShopSettingsUpdate($shopSettingsInput: ShopSettingsInput!) {
+              shopSettingsUpdate(input: $shopSettingsInput) {
+                shopErrors {
+                        field
+                        message
+                        code
+                    }
+              }
+            }
+        """
+
+        response = graphql_request(
+            query, variables, self.headers, self.endpoint_url)
+
+        errors = response["data"]["shopSettingsUpdate"]["shopErrors"]
+        handle_errors(errors)
+
```

Comment:
I think we should return an ID or something as a confirmation that the mutation worked

Reply:
right, but shop doesn't have ID if I'm not wrong
we can return the shop object with updated fields

Reply:
yes perfect